webapp/channels/src/components/advanced_text_editor/send_button/scheduled_post_custom_time_modal/scheduled_post_custom_time_modal.scss

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+.GenericModal__body {
```

Comment:
Why define these styles twice? From what I can see here, they should only be needed on the `DeleteScheduledPostModal`, but if they are needed for both, I think they should either be moved to some shared component or made narrower. As-is, they're applied to every `GenericModal` in the app which seems like it's likely to affect another modal accidentally

Reply:
Actually, I've now done something even better. There's actually an existing field called `errorText` in generic modal, so now I'm using that to display the error instead of a custom code. So this is now gone.